### PR TITLE
[MTDSA-8814] Play 2.8 upgrade

### PR DESCRIPTION
**Pull request by james-work-account**

### CODE REVIEW

File: project/build.properties [hunk 1]
```diff
@@ -1 +1 @@
-sbt.version=1.3.13
+sbt.version=1.5.0
```
Comment (Thepworth95) [inline project/build.properties:1]: can be bumped to 1.5.2

File: scalastyle-confg.xml [hunk 2]
```diff
@@ -0,0 +1,117 @@
+<scalastyle>
```
Comment (Thepworth95) [inline scalastyle-confg.xml:1]: should this be in the git ignore? presume its been added by one of the dependancies

Comment (james-work-account) [reply scalastyle-confg.xml:1]: This enforces things like file headings, it's something we have in other repos but wasn't in this.

File: conf/application.conf [hunk 3]
```diff
@@ -116,7 +116,7 @@ api {
 # Allowlisted headers
```
Comment (BetaDraconis) [inline conf/application.conf:116]: here

**Review by BetaDraconis — APPROVED** [review]

**Pull request merged**